tests/test_closing.py

```diff
@@ -0,0 +1,20 @@
+"""Tests for Report Generation functions."""
+# Standard Python Libraries
+import types
+
+# cisagov Libraries
+from pca_report_library.customer.closing import deltas, pairwise
+
+
+def test_deltas_returns_generator():
+    """Test generator type is returned."""
+    in_val = [1, 2, 3, 4]
+    result = deltas(in_val)
+    assert isinstance(result, types.GeneratorType)
+
+
+def test_pairwise_returns_zip():
+    """Test zip type is returned."""
+    in_val = [1, 2, 3, 4]
+    result = pairwise(iter(in_val))
+    assert isinstance(result, zip)
```

Comment:
Please create an issue in GitHub to flesh out the tests for this repository so that they test the actual code.

Reply:
Hi Shane, the GitHub issue has been created here: https://github.com/cisagov/pca-report-library/issues/11 
Please let me know if anything should be added to it, thanks.